service/history/queues/executable.go

```diff
@@ -476,6 +476,16 @@ func (e *executableImpl) isExpectedRetryableError(err error) (isRetryable bool,
 		return true, consts.ErrNamespaceHandover
 	}
 
+	// Check for serviceerror.Aborted (shard ownership lost, etc.)
```

Comment:
Changing the retry behavior for all task processing without understanding the implications is dangerous.

Reply:
removed